application.fam

```diff
@@ -9,7 +9,7 @@ App(
     requires=[
         "gui", "storage", "nfc",
     ],
-    stack_size=7 * 1024,
+    stack_size=5 * 1024,
```

Comment:
was getting out of memory problems.  I'm starting to think the threads inside the app that alloc memory are doing so on the heap, or at least separate from this.